File: src/main/java/kitchenpos/menus/tobe/menugroup/infra/JpaMenuGroupRepositoryV2.java
package kitchenpos.menus.tobe.menugroup.infra;

import java.util.UUID;
import kitchenpos.menus.tobe.menugroup.domain.model.MenuGroupV2;
import kitchenpos.menus.tobe.menugroup.domain.repository.MenuGroupRepositoryV2;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface JpaMenuGroupRepositoryV2 extends MenuGroupRepositoryV2, JpaRepository<MenuGroupV2, UUID> {
}

File: src/main/kotlin/kitchenpos/tobe/menu/infra/JpaMenuGroupRepositoryV2.kt
package kitchenpos.tobe.menu.infra

import kitchenpos.tobe.menu.domain.entity.MenuGroupV2
import kitchenpos.tobe.menu.domain.repository.MenuGroupRepositoryV2
import org.springframework.data.jpa.repository.JpaRepository
import java.util.*

interface JpaMenuGroupRepositoryV2 : JpaRepository<MenuGroupV2, UUID>, MenuGroupRepositoryV2 {
    override fun findMenuGroupById(id: UUID): MenuGroupV2?
}
